docs/features.md

```diff
@@ -0,0 +1,37 @@
+# Pypiper features at-a-glance
```

Comment:
What's this markdown syntax do? Are the empty brackets allowed and just used when no naming is needed?

Reply:
those are images without `alt` tags. just lazy I guess.

Reply:
Cool. I don't know much about Markdown syntax so wasn't sure.